internal/commands/publish.go

```diff
@@ -374,35 +384,41 @@ func endOfSupportFor(publishDate time.Time) string {
 	return endOfNinthMonth.Format(publishDateFormat)
 }
 
-func (p Publish) attachLicenseFile(slug string, releaseID int, version *releaseVersion) (string, error) {
-	if version.IsGA() {
-		productFiles, err := p.PivnetProductFilesService.List(slug)
-		if err != nil {
-			return "", err
-		}
+func (p Publish) attachLicenseFiles(slug string, releaseID int, version *releaseVersion) ([]string, error) {
+	var attachedLicenseFileNames []string
 
-		productFile, found := findMatchingOSL(productFiles, version)
-		if !found {
-			return "", errors.New("required license file doesn't exist on Pivnet")
-		}
+	if !version.IsGA() {
+		return []string{}, nil
```

Comment:
you can `return nil, nil`. Slices are fancy pointers so this does more than what's needed. It not a big deal in this case to return `[]string{}` but it adds an allocation and is more code than needed.

[See this comment for more details](https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices)

Reply:
Awesome, thanks for the context on this!  i've made that change and pushed it up, i'm not sure if it'll retrigger the code review, but if not, i'll just merge it in once the checks pass.

Thanks again for all the review on this PR, Christopher!